README.md

```diff
@@ -10,6 +10,10 @@
 
 The easiest way to get a production grade Kubernetes cluster up and running.
 
+## 2020-05-06 etcd-manager Certificate Expiration Advisory
+
+kops versions released today contain a **critical update** to etcd-manager: 1 year after creation (or first adopting etcd-manager), clusters will stop responding due to expiration of a TLS certificate. Upgrading kops to 1.15.3, 1.16.2, 1.17.0-beta.2, or 1.18.0-alpha.3 is highly recommended. Please see the [advisory](./docs/advisories/etcd-manager-certificate-expiration.md) for the full details.
```

Comment:
Looks good.  Should we call it a "critical fix" or a "critical update fix"?  I just worry that a skim-read might suggest it's not now fixed....